rct229/rulesets/ashrae9012019/section19/section19rule18.py

```diff
@@ -0,0 +1,366 @@
+                        or (total_fan_power_b < expected_fan_wattage_b)
+                    )
+                )
+                or (
+                    not more_than_one_supply_fan_b
+                    and total_fan_power_b < expected_fan_wattage_b
+                )
+            )
+
+        def get_manual_check_required_msg(self, context, calc_vals=None, data=None):
+            hvac_b = context.BASELINE_0
+            hvac_id_b = hvac_b["id"]
+
+            more_than_one_exhaust_fan_and_energy_rec_is_relevant_b = calc_vals[
+                "more_than_one_exhaust_fan_and_energy_rec_is_relevant_b"
+            ]
+            more_than_one_supply_fan_b = calc_vals["more_than_one_supply_fan_b"]
+            expected_fan_wattage_b = calc_vals["expected_fan_wattage_b"]
+
+            black_word = (
+                ""
+                if more_than_one_exhaust_fan_and_energy_rec_is_relevant_b
+                else "energy recovery"
+            )
+
+            undetermined_msg = ""
+            if (
+                more_than_one_supply_fan_b
+                and more_than_one_exhaust_fan_and_energy_rec_is_relevant_b
+            ):
+                undetermined_msg = (
+                    f"{hvac_id_b} has more than one supply fan and/or more than one exhaust fan associated with the HVAC system in the baseline and "
+                    f"therefore this check could not be conducted for this HVAC system. Conduct manual check for compliance with G3.1.2.9."
+                )
+            else:
+                undetermined_msg = (
+                    f"Fan power for {hvac_id_b} is greater than expected per Section and Table G3.1.2.9 assuming no pressure drop adjustments (e.g., sound attenuation, air filtration, fully ducted return "
+                    f"when required by code, airflow control devices, carbon and other gas-phase air cleaners, coil runaround loops, evaporative humidifier/coolers in series with another cooling coil, "
+                    f"exhaust systems serving fume hoods, and laboratory and vivarium exhaust systems in high-rise buildings) per Table 6.5.3.1-2 other than {black_word} and "
+                    f"MERV filters defined in the RMD (if modeled). Expected Wattage = {expected_fan_wattage_b.to(ureg.kW)} kW. however not all pressure drop adjustments are able to be captured in the RMD so conduct manual check."
+                )
+
+            return undetermined_msg
+
+        def rule_check(self, context, calc_vals=None, data=None):
+            more_than_one_exhaust_fan_and_energy_rec_is_relevant_b = calc_vals[
+                "more_than_one_exhaust_fan_and_energy_rec_is_relevant_b"
+            ]
+            more_than_one_supply_fan_b = calc_vals["more_than_one_supply_fan_b"]
+            total_fan_power_b = calc_vals["total_fan_power_b"]
+            expected_fan_wattage_b = calc_vals["expected_fan_wattage_b"]
+
+            return not more_than_one_exhaust_fan_and_energy_rec_is_relevant_b and (
```

Comment:
For the above reasoning, we do not need to consider the exhaust fan energy anymore. So in here, we should just focus on whether `total_fan_power_b == expected_fan_wattage_b`

Reply:
Addressed